lib/gui/components/svg-icon/svg-icon.js

```diff
@@ -21,8 +21,94 @@
  */
 
 const angular = require('angular');
+const react = require('react');
+const propTypes = require('prop-types');
+const react2angular = require('react2angular').react2angular;
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
+
 const MODULE_NAME = 'Etcher.Components.SVGIcon';
-const SVGIcon = angular.module(MODULE_NAME, []);
-SVGIcon.directive('svgIcon', require('./directives/svg-icon'));
+const angularSVGIcon = angular.module(MODULE_NAME, []);
+
+/**
+ * @summary SVG element that takes both filepaths and file contents
+ * @type {Object}
+ * @public
+ */
+class SVGIcon extends react.Component {
+
+  /**
+   * @summary Render the SVG
+   * @returns {react.Element}
+   */
+  render() {
+    let contents = null;
+
+    // The path contains SVG contents
+    if (_.first(this.props.path) === '<') {
```

Comment:
It's functionality that was inherited from the Angular svg-icon directive, but we're not making use of inlining SVG contents, so we could remove it. I can't really think of a use-case where it's absolutely necessary—if we want to dynamically generate SVGs it might be better to just allow passing elements.